### PR TITLE
split resource and update manpage

### PR DESCRIPTION
**Pull request by Aym-fuhong**

### CODE REVIEW

**Comment by seafruit** [comment]
:+1:

**Comment by goddessting** [comment]
:+1:

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "description": "Delicious School =================",
   "main": "index.js",
   "scripts": {
+    "init-db":"babel-node setup/init-db.js",
```
Comment (goddessting) [inline package.json:7]: 格式化

File: public/meal-info.jsx [hunk 2]
```diff
@@ -52,47 +62,78 @@ export default class MealInfo extends Component {
...
-      if (count <= 0) {
-        count = 1;
-      }
-      $('.count').html(count);
-      // this.setState.count=count;
-    }
+  getDishInformation() {
+    const url = location.href;
+    let id = url.split('=')[1].split('&')[0];
+    request.post('/api/mealInfo')
```
Comment (goddessting) [inline public/meal-info.jsx:80]: request.get('/api/dishes/id')

File: public/meal-info.jsx [hunk 3]
```diff
@@ -52,47 +62,78 @@ export default class MealInfo extends Component {
...
+      .send({id: id})
```
Comment (goddessting) [inline public/meal-info.jsx:81]: delete

File: public/meal-info.jsx [hunk 4]
```diff
@@ -52,47 +62,78 @@ export default class MealInfo extends Component {
...
+        const mealInfo = res.body;
+        if (err) return alert(err);
+        this.setState({
+          mealInfo: mealInfo,
+        });
+      });
   }
 
-  myOrder(dishname, dishprice, dishstore, count) {
+  myOrder(dishname, dishprice, storename, storephone, storelocation, count) {
```
Comment (goddessting) [inline public/meal-info.jsx]: nnnnnnnnnnNmmmm

**Pull request merged**